patternlab/styleguide/source/assets/js/modules/mainNavMixed.js

```diff
@@ -1,293 +1,27 @@
-      
-      switch (e.code) {
-        case 'ArrowUp':
-        case 'ArrowLeft':
-          if (subItem = prevSib) {
-            prevSib.querySelector('.js-main-nav-hamburger__link').focus();
-          }
-          break;
-        case 'ArrowDown':
-        case 'ArrowRight':
-          if (subItem = nextSib) {
-            nextSib.querySelector('.js-main-nav-hamburger__link').focus();
-          }
-          break;
-        case 'Escape':
-          itemButton.focus();
-
-           break;
-      }
-    });
-  });
-});
-
-function closeMenu() {
-  body.classList.remove("show-menu-hamburger");
-  menuOverlay.classList.remove("overlay-open");
-  menuButtonText.textContent = "Menu";
-  menuButton.setAttribute('aria-pressed', 'false');
-  menuButton.setAttribute('aria-expanded', 'false');
-  menuButton.setAttribute('aria-label', 'Open the main menu for mass.gov');
-  buttonLabel = "Menu";
-  feedbackButton.classList.remove('hide-button');
-}
-
-function openMenu() {
-  body.classList.add("show-menu-hamburger");
-  menuOverlay.classList.add("overlay-open");
-  menuButtonText.textContent = "Close";
-  buttonLabel = "Close";
-  menuButton.setAttribute('aria-pressed', 'true');
-  menuButton.setAttribute('aria-expanded', 'true');
-  menuButton.setAttribute('aria-label', 'Close the main menu for mass.gov');
-  feedbackButton.classList.add('hide-button');
-}
-
-function closeSubMenus(item) {
-  //Close other open submenus
-  let siblings = [];
-  let thisSibling = item.parentNode.firstChild;
-
-  while (thisSibling) {
-    if (thisSibling !== item && thisSibling.nodeType === Node.ELEMENT_NODE)
-      siblings.push(thisSibling);
-    thisSibling = thisSibling.nextElementSibling || thisSibling.nextSibling;
   }
-
-  for (let i = 0; i < siblings.length; i++) {
-    if (siblings[i].classList.contains('submenu-open')) {
-
-      setTimeout(function timeoutFunction() {
-        siblings[i].querySelector('.js-main-nav-hamburger-content').style.height = "0";
-        siblings[i].querySelector('.js-main-nav-hamburger-content').classList.add('is-closed');
-        siblings[i].querySelector('.js-main-nav-hamburger__container').style.opacity = "0";
-        siblings[i].classList.remove('submenu-open');
-        siblings[i].querySelector('.js-main-nav-hamburger__top-link').setAttribute('aria-expanded', 'false');
-        siblings[i].querySelector('.js-main-nav-hamburger__top-link').setAttribute('aria-label', 'show menu');
-        
-      }, 500);
-
-    }
-  }
-}
-
-// Close menu when utility nav is clicked
-if (null !== utilNavWide) {
-  utilNavWide.addEventListener('click', function () {
-    closeMenu();
-  });
-}
-
-
-// Close and reset menu on overlay click
-if (null !== menuOverlay) {
-  menuOverlay.addEventListener("click", function () {
-    closeMenu();
-  });
-}
-
-let debouncer;
-window.onresize = function () {
-  clearTimeout(debouncer);
-  debouncer = setTimeout(function () {
-    width = body.clientWidth;
-  }, 100);
-};
+}
```

Comment:
Let's add a newline to the end of the file

Reply:
A new line is added at the end of the file.